logging/kit/adapter.go

```diff
@@ -47,6 +47,13 @@ func (a *adapter) Info(msg string, data ...interface{}) {
 	a.Logger.Log(ctx...)
 }
 
+// Info logs warning messages using go-kit.
+func (a *adapter) Warn(msg string, data ...interface{}) {
+	ctx := []interface{}{"lvl", "warn", "msg", msg}
+	ctx = append(ctx, data...)
+	a.Logger.Log(ctx...)
```

Comment:
I want to ignore this warning, because other same methods also were not checked.
https://github.com/shogo82148/goa-v1/blob/346ea2b179d06f908ab7f55b39771d2663936a6a/logging/kit/adapter.go#L47